integration_tests/run.sh

```diff
@@ -1,37 +1,58 @@
 #!/bin/bash
 # Script for jenkins to run tests on polar2grid.
 
-set -exf
+set -ex
 export PATH="/usr/local/texlive/2019/bin/x86_64-linux":$PATH
 cd "$WORKSPACE"
 # Activate conda for bash.
 /data/users/davidh/miniconda3/bin/conda init bash
 # Restart the shell to enable conda.
 source ~/.bashrc
-conda env update -n jenkins_p2g_env -f build_environment.yml --prune
-conda activate jenkins_p2g_env
+conda env update -n jenkins_p2g_swbundle -f build_environment.yml
+conda activate jenkins_p2g_swbundle
 
-time=`date +"%Y%m%d-%H%M%S"`
-tarball_name=polar2grid-swbundle-$time
-./create_conda_software_bundle.sh $WORKSPACE/$tarball_name
-export POLAR2GRID_HOME="$WORKSPACE/$tarball_name"
+# Handle release vs test naming.
+prefix="$(cut -d'-' -f1 <<<"$GIT_TAG_NAME")"
+end="`date +"%Y%m%d-%H%M%S"`"
+# If the tag is correct and a version was specified, make a version release.
+if [[ "$GIT_TAG_NAME" =~ [pg]2g-v[0-9]+\.[0-9]+\.[0-9]* ]]; then
+    # Removes prefix from $GIT_TAG_NAME.
+    end=${GIT_TAG_NAME#"$prefix-"}
+fi
+if [[ "$prefix" = "g2g" ]]; then
+    prefix=geo
+else
+    prefix=polar
+fi
+swbundle_name=${prefix}2grid-swbundle-${end}
+
+./create_conda_software_bundle.sh ${WORKSPACE}/${swbundle_name}
```

Comment:
*Iff* you have to make other changes, could you wrap this path in quotes. If either of the variables have spaces in them then they will be treated as separate arguments (99% sure).